utils/Cargo.toml

```diff
@@ -46,24 +46,24 @@ nimiq-keys = { path = "../keys" }
 [features]
 crc = []
 otp = ["beserial", "clear_on_drop", "nimiq-hash", "rand"]
-key-store = ["thiserror", "beserial"]
+key-store = ["beserial", "log", "thiserror"]
```

Comment:
Most of the changes in this file is to alphabetically order the dependencies but note that here I'm actually adding `log` to the `key-store` feature.